### ISSUE TITLE
readonly 

### ISSUE THREAD

**Issue by kinden** [crocodic-studio/crudbooster#529]
without using child form.....

i am getting only same value that is first value. actually i should get particular value when i click in particular row. 

![6](https://cloud.githubusercontent.com/assets/27999833/26041745/290a9ac8-3951-11e7-88fe-21c712238d70.PNG)

![5](https://cloud.githubusercontent.com/assets/27999833/26041693/b50ed756-3950-11e7-8305-1e6f15439e00.PNG)

code:

$Name = DB::table('patient')->value('Name');........(i don;t know how i can get with id)
$CID = DB::table('patient')->value('CID');.................(i don;t know how i can get with id)
			
$this->form = [];$this->form[] = ['label'=>'CID', 'name'=> 'CID','type'=>'number','width'=>'col-sm-6','readonly'=>true,'value'=>$CID];
	
$this->form[] = ['label'=>'Name','name'=>'Name','type'=>'text','width'=>'col-sm 6','readonly'=>true,'value'=>$Name];
			

please help me

**Comment by kinden** [comment]
Hi.... FherryFherry, please help me to solved problem about mention ...

**Comment by fherryfherry** [comment]
I don't really understand what do you want, 
Do you want to put patient CID to visit form?

first, you need to pass the id to add URL.
`$this->addaction[] = ['label'=>'Set Active','url'=>CRUDBooster::adminPath('visit/add?id_patient=[id]'),'icon'=>'fa fa-check','color'=>'success'];`

then in the visit controller, you need to get the patient data:
```php
//Get the patient data, where id equal with get id patient
$patient = DB::table('patient')->where('id',Request::get('id_patient'))->first();

//Put some value to the add form
if(CRUDBooster::getCurrentMethod() == 'getAdd') {
$this->form[] = ['label'=>'CID','type'=>'text','name'=>'cid','value'=>$patient->cid,'readonly'=>true];
}else {
$this->form[] = ['label'=>'CID','type'=>'text','name'=>'cid','readonly'=>true];
}
```

**Comment by kinden** [comment]
yes,,, but how i can hide the datamodal  form field.....its showing this field.. actually i have 2 ways to do. this datamodal form field is doing directly from visit manu. 
![444](https://cloud.githubusercontent.com/assets/27999833/26245103/77c02b22-3cb4-11e7-82ad-7e505acb96db.PNG)

**Comment by fherryfherry** [comment]
Again, you can use if condition with CRUDBooster::getCurrentMethod()

**Comment by kinden** [comment]
can you give me code,  i m new in laravel....FherryFherry... Your crudbooster helps  a lot to me...Thanks in advance

**Comment by kinden** [comment]
Hi FherryFherr,,,,,how are you hope doing well,,,,,,,,and How to hide above datamodal form filed that is browse data when  we click on addaction button....

**Comment by kinden** [comment]
Below is datamodal code:

$this->form[] = ['label'=>'Select Name','name'=>'Patient_Register_ID','type'=>'datamodal','readonly'=>true,'width'=>'col-sm-6','datamodal_table'=>'patient','datamodal_columns'=>'Name,CID,Contact','datamodal_columns_alias'=>'Name,CID,Contact No','datamodal_select_to'=>'Name:Name,CID:CID','datamodal_size'=>'small'];

**Comment by fherryfherry** [comment]
I mean you can modify like bellow:
```php
//Get the patient data, where id equal with get id patient
$patient = DB::table('patient')->where('id',Request::get('id_patient'))->first();

//Put some value to the add form
if(CRUDBooster::getCurrentMethod() == 'getAdd') {
$this->form[] = ['label'=>'CID','type'=>'text','name'=>'cid','value'=>$patient->cid,'readonly'=>true];
}else {
$this->form[] = ['label'=>'CID','type'=>'text','name'=>'cid','readonly'=>true];
}

if(!Request::get('id_patient')) {
$this->form[] = ['label'=>'Select Name','name'=>'Patient_Register_ID','type'=>'datamodal','readonly'=>true,'width'=>'col-sm-6','datamodal_table'=>'patient','datamodal_columns'=>'Name,CID,Contact','datamodal_columns_alias'=>'Name,CID,Contact No','datamodal_select_to'=>'Name:Name,CID:CID','datamodal_size'=>'small'];
}
```

**Comment by kinden** [comment]
Thanks fherryfherry... now i got what i want.....
